composer.json

```diff
@@ -93,27 +91,32 @@
         "prestashop/statssearch": "^2",
         "prestashop/statsstock": "^2",
         "prestashop/statsvisits": "^2",
-        "prestashop/translationtools-bundle": "^3",
+        "prestashop/translationtools-bundle": "dev-sf4-support",
```

Comment:
https://github.com/PrestaShop/PrestaShop/pull/13761/files#r283138318

Reply:
Because waiting for TranslationToolsBundle pr to be merged

Reply:
done, solved!